### ISSUE TITLE
Per-pixel confidence loss question

### ISSUE THREAD

**Issue by phongnhhn92** [shaohua0116/Multiview2Novelview#4]
Hello, 
In the equation (5) in your paper, there is a element-wise square operator like this:
![eq5](https://user-images.githubusercontent.com/11288381/49963945-02751a80-ff22-11e8-9305-7daf3eff1a48.png)
I believe this loss function was defined in this piece of code
`l1_loss += tf.reduce_mean(loss_map * normalized_mask) * current_weight / \
(int(img.get_shape()[1]) * int(img.get_shape()[2])) * regularizer_weight`

I wonder it should be like this
`l1_loss += tf.reduce_mean((loss_map**2) * normalized_mask) * current_weight / \
(int(img.get_shape()[1]) * int(img.get_shape()[2])) * regularizer_weight`

Can u verify it is correct ? If your code is correct then I am sorry for my question. I just confused when I see the loss like that. Thanks !
